_includes/momentumstats-carousel.html

```diff
@@ -0,0 +1,46 @@
+{% assign mstats = site.momentumstats | where: "display", "true" %}
+
+{% if mstats.size > 0 %}
+	<style type="text/css">
+		img.momentumstat-image
+		{
+			width: 420px;
+		}
+		
+	</style>
+
+	<h3>
+		Check out some of our momentum!
+	</h3>
+
+	<div class="momentumstats-carousel-container">
+
+		{% for momentumstat in mstats %}
+			<div class="momentumstats-slide-container" id="momentumslide{{ forloop.index0 }}"
+				{% if forloop.first %}
+					style="display: inline;"
+				{% else %}
+					style="display: none;"
+				{% endif %}
+				>
+				{% include momentumstats-carousel-slide.html momentumstat=momentumstat %}
+			</div>
+		{% endfor %}
+	</div>
+
+	<script language='javascript'>
+		setInterval(function() {
+			thingies = jQuery(".momentumstats-slide-container");
```

Comment:
Written as it is  here the `thingies` symbol will be attached implicitly as a property of `window`. Is this the intention, or was omitting `const`, `var`, or `let` accidental? Note that if an eventual change was put in place that put global scope into Strict Mode that this would throw a `TypeError` exception.

Reply:
Gotcha. Nope, this was me sacrificing quality for something that would function. I can refactor to avoid the TypeError.